### PR TITLE
feat: Quickstart with docker-compose

### PR DESCRIPTION
**Pull request by andpopov**
- docker-compose file to up Artipie backend and frontend
- wiki page with docker-compose's quickstart

### CODE REVIEW

**Comment by andpopov** [comment]
@olenagerasimova please review

File: .wiki/DockerCompose.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Quickstart with docker-compose
+
+Make sure you have already installed both [Docker Engine](https://docs.docker.com/get-docker/) and
+[Docker Compose](https://docs.docker.com/compose/install/).
+Then, obtain `docker-compose.yaml` file from the repository:
+you can [open it from the browser](https://github.com/artipie/artipie/blob/master/docker-compose.yaml),
+copy content and save it locally or use [git](https://git-scm.com/) and [clone](https://git-scm.com/docs/git-clone) the repository.
+As soon as Docker Compose is installed and `docker-compose.yaml` file is retrieved, open command line,
```
Comment (olenagerasimova) [inline .wiki/DockerCompose.md:8]: "As soon as Docker Compose is installed" - this part seems a bit off, as in the first sentence we already demanded Docker Compose to be installed.

File: .wiki/DockerCompose.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+`cd` to the location with the compose file and run Artipie service:
+
+```bash
+docker-compose up
+```
+
+It'll start a new Docker containers with latest Artipie and Artipie dashboard service images.
```
Comment (olenagerasimova) [inline .wiki/DockerCompose.md:15]: "It'll start a new Docker containers" - "a" cannot be used with plural, so I suggest to remove this "a"

File: .wiki/DockerCompose.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+A new Artipie's container generates default configuration if not found at `/etc/artipie/artipie.yml`,
```
Comment (olenagerasimova) [inline .wiki/DockerCompose.md:16]: here "A" is also not needed

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@andpopov thanks, added several minor comments

File: .wiki/DockerCompose.md (hunk 3 above)
Comment (andpopov) [reply .wiki/DockerCompose.md]: Thanks, corrected.

**Comment by olenagerasimova** [comment]
@andpopov thanks, looks good

**Review by olenagerasimova — APPROVED** [review]

**Pull request merged**